### PR TITLE
Make `handle_options` public again.

### PR DESCRIPTION
**Pull request by nnethercote**
r? @compiler-errors

### CODE REVIEW

**Comment by nnethercote** [comment]
I added `<>` to the URL.

@bors r=compiler-errors

File: compiler/rustc_driver_impl/src/lib.rs [hunk 1]
```diff
@@ -1184,7 +1184,11 @@ fn print_flag_list<T>(
 ///
 /// So with all that in mind, the comments below have some more detail about the
 /// contortions done here to get things to work out correctly.
-fn handle_options(handler: &EarlyErrorHandler, args: &[String]) -> Option<getopts::Matches> {
+///
+/// This does not need to be `pub` for rustc itself, but @chaosite needs it to
+/// be public when using rustc as a library, see
+/// <https://github.com/rust-lang/rust/commit/2b4c33817a5aaecabf4c6598d41e190080ec119e>
```
Comment (bjorn3) [inline compiler/rustc_driver_impl/src/lib.rs:1190]: This seems like it is the wrong commit. I don't see any where that this person requested the function to remain public. Neither on that commit itself, nor on the associated PR.

Comment (bjorn3) [reply compiler/rustc_driver_impl/src/lib.rs:1190]: @nnethercote do you remember what the correct commit is?

Comment (nnethercote) [reply compiler/rustc_driver_impl/src/lib.rs:1190]: It's the right commit. There is a comment added to that commit after it merged, but it's easy to overlook in the GitHub UI. [This link](
https://github.com/rust-lang/rust/commit/2b4c33817a5aaecabf4c6598d41e190080ec119e#diff-39877a2556ea309c89384956740d5892a59cef024aa9473cce16bbdd99287937R1187) should be better, it links directly to the comment.

**Pull request merged**